plutus-core-spec/plutus-core-specification.tex

```diff
@@ -239,6 +240,13 @@ \section{Plutus Core}
 to allow us to use standard combinators (for example the $Z$
 combinator and Church/Scott encoded booleans) for such purposes.
 
+\paragraph{Implementation.} A compiler and evaluator for Plutus Core can be found at
+\url{https://github.com/input-output-hk/plutus}.
```

Comment:
I think we probably mention this enough times that a citation entry for the plutus repo would make sense.

Reply:
I turned the GitHub links into references, but I'm not totally convinced. I'm never sure what the best way to refer to things that only exist on-line is.  It's not a big deal though.